index.bs

```diff
@@ -521,16 +522,22 @@ When this method is invoked, the user agent MUST execute the following algorithm
     cancel the timer started in step 2, return a {{DOMException}} whose name is "{{NotSupportedError}}", and terminate this
     algorithm.
 
-1. Let |clientExtensions| be a new [=list=].
+1. Let |clientExtensions| be a new [=list=] and let |authenticatorExtensions| be a new [=list=].
 
 1. If the {{MakeCredentialOptions/extensions}} member of |options| is [=present=], then [=map/for each=]
-    |extension| → |argument| of <code>|options|.{{MakeCredentialOptions/extensions}}</code>:
-    1. If |extension| is not supported by this client platform, then [=continue=].
+    |extensionId| → |clientExtensionInput| of <code>|options|.{{MakeCredentialOptions/extensions}}</code>:
+    1. If |extensionId| is not supported by this client platform or is not a [=registration extension=], then [=continue=].
 
-    1. Otherwise, let |result| be the result of running |extension|'s [=client extension processing=] algorithm on |argument|. If the
+    1. [=list/Append=] the entry |extensionId| → |clientExtensionInput| to |clientExtensions|.
+
+    1. If |extensionId| is not an [=authenticator extension=], then [=continue=].
+
+    1. Let |result| be the base64url encoded representation of 
+        the result of running |extensionId|'s [=client extension processing=] algorithm
```

Comment:
To base64url encode something, it needs to be an array of bytes, but I don't see a restriction in https://api.csswg.org/bikeshed/?url=https://raw.githubusercontent.com/selfissued/webauthn/mbj-extensions-connective-tissue/index.bs#sctn-client-extension-processing that all extensions return arrays of bytes from their client extension processing. [appid](https://api.csswg.org/bikeshed/?url=https://raw.githubusercontent.com/selfissued/webauthn/mbj-extensions-connective-tissue/index.bs#sctn-appid-extension) seems not to return anything, for example.